addons/website_event_booth/controllers/event_booth.py

```diff
@@ -79,47 +79,26 @@ def _get_requested_booths(self, event, event_booth_ids):
             ('state', '=', 'available'),
             ('id', 'in', booth_ids)
         ])
-        if booth_ids != booths.ids:
-            raise Forbidden(_('Booth registration failed. Please try again.'))
-        if len(booths.booth_category_id) != 1:
-            raise Forbidden(_('Booths should belong to the same category.'))
+        if booth_ids != booths.ids or len(booths.booth_category_id) != 1:
+            return False
         return booths
 
     @http.route('/event/<model("event.event"):event>/booth/confirm',
                 type='http', auth='public', methods=['POST'], website=True, sitemap=False)
     def event_booth_registration_confirm(self, event, booth_category_id, event_booth_ids, **kwargs):
         booths = self._get_requested_booths(event, event_booth_ids)
 
-        booth_values = self._prepare_booth_registration_values(event, kwargs)
-        booths.action_confirm(booth_values)
-
-        return request.redirect(('/event/%s/booth/success?' % event.id) + werkzeug.urls.url_encode({
-            'booths': ','.join([str(id) for id in booths.ids]),
-        }))
-
-    # This will be removed soon
-    @http.route('/event/<model("event.event"):event>/booth/success',
-                type='http', auth='public', methods=['GET'], website=True, sitemap=False)
-    def event_booth_registration_complete(self, event, booths):
-        booth_ids = request.env['event.booth'].sudo().search([
-            ('event_id', '=', event.id),
-            ('state', '=', 'unavailable'),
-            ('id', 'in', [int(id) for id in booths.split(',')]),
-        ])
-        if len(booth_ids.mapped('partner_id')) > 1:
-            raise NotFound()
-        event_sudo = event.sudo()
-        return request.render(
-            'website_event_booth.event_booth_registration_complete',
-            {'event': event,
-             'event_booths': event_sudo.event_booth_ids,
-             'main_object': event,
-             'contact_name': booth_ids[0].contact_name or booth_ids.partner_id.name,
-             'contact_email': booth_ids[0].contact_email or booth_ids.partner_id.email,
-             'contact_mobile': booth_ids[0].contact_mobile or booth_ids.partner_id.mobile,
-             'contact_phone': booth_ids[0].contact_phone or booth_ids.partner_id.phone,
-             }
-        )
+        if booths:
+            booth_values = self._prepare_booth_registration_values(event, kwargs)
+            booths.action_confirm(booth_values)
+
+            return json.dumps({
+                'success': True,
+                'event_name': event.name,
+                'contact': {
+                    key.partition('contact_')[2]: val for key, val in booth_values.items() if key.startswith('contact_')},
```

Comment:
Linting: you may multi-line to ease reading :)
```
    'contact': {
        key.partition('contact_')[2]: val
        for key, val in booth_values.items()
        if key.startswith('contact_')
    },
```
By the way why not doing it manually for the 4 fields ? Do we have inheritance adding some extra fields ?